src/tests/unit/tests/reports/report-html-generator.test.tsx

```diff
@@ -40,11 +40,11 @@ describe('ReportHtmlGenerator', () => {
         const getGuidanceTagsStub: GetGuidanceTagsFromGuidanceLinks = () => [];
 
         const sectionFactoryMock = Mock.ofType<ReportSectionFactory>();
-        const environmentInfo: EnvironmentInfo = {
-            axeCoreVersion,
-            browserSpec,
+        const environmentInfoProvider: EnvironmentInfoProvider = new EnvironmentInfoProvider(
```

Comment:
Same here as in the other test: this should be a mock, not an instance of the actual class 'FooterText' has a dependency on.